backend/open_webui/test/apps/webui/storage/test_provider.py

```
@@ -0,0 +1,274 @@
+
+
+def mock_upload_dir(monkeypatch, tmp_path):
+    """Fixture to monkey-patch the UPLOAD_DIR and create a temporary directory."""
+    directory = tmp_path / "uploads"
+    directory.mkdir()
+    monkeypatch.setattr(provider, "UPLOAD_DIR", str(directory))
+    return directory
+
+
+def test_imports():
+    provider.StorageProvider
```

Ruff (0.8.2)

Found useless expression. Either assign it to a variable or remove it.

(B018)